chat-plugins/info.js

```diff
@@ -1140,6 +1140,27 @@ exports.commands = {
 		this.sendReplyBox(`<button name="openSounds" class="button"><i style="font-size: 16px; vertical-align: -1px" class="fa fa-volume-up"></i> Sound</button> (The Sound and Options buttons are at the top right, next to your username)`);
 	},
 
+	'!sanction': true,
+	punishment: 'sanction',
+	sanction: function (target, rom, user) {
```

Comment:
I would generally call this `punishments`. "sanction" is too obscure of a word for here.